lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -39,6 +39,21 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
         this.logger.verbose("initializeAuthorizationRequest called", request.correlationId);
         const generatedPkceParams = await this.browserCrypto.generatePkceCodes();
 
+        // Generate Session Transport Key if Refresh Token Binding is enabled
+        if (this.config.auth.refreshTokenBinding) {
+            this.logger.verbose("Refresh token binding enabled, attempting to generate Session Transport Key");
+            try {
+                const sessionTransportKeyThumbprint = await this.browserCrypto.getPublicKeyThumbprint(request, CryptoKeyTypes.stk_jwk);
+                request.stkJwk = sessionTransportKeyThumbprint;
+                this.logger.verbose("Successfully generated and stored Session Transport Key");
+            } catch(error) {
+                if(error instanceof BrowserAuthError) {
+                    this.logger.error(error.message);
```

Comment:
I don't think you should log the message as-is, instead write a hard-coded message and maybe pull off certain fields such as the errorCode.